modules/web-test/variables.tf

```diff
@@ -0,0 +1,115 @@
+variable "resource_group_name" {
+  description = "The name of the resource group to create the resources in."
+  type        = string
+}
+
+variable "location" {
+  description = "The location to create the resources in."
+  type        = string
+}
+
+variable "component_id" {
+  description = "The Application Insights component ID."
+  type        = string
+}
+
+variable "name" {
+  description = "The name of this Web Test."
+  type        = string
+}
+
+variable "configuration" {
+  description = "An XML configuration specification for this Web Test."
+  type        = string
+  default     = null
+}
+
+variable "kind" {
+  description = "The kind of web test that this Web Test watches."
+  type        = string
+  default     = "standard"
+
+  validation {
+    condition     = contains(["standard", "ping", "multistep"], var.kind)
+    error_message = "Web Test kind must be \"standard\", \"ping\" or \"multistep\"."
+  }
+}
+
+variable "description" {
+  description = "The purpose/user defined description of this Web Test."
+  type        = string
+  default     = null
+}
+
+variable "enabled" {
+  description = "Is this Web Test enabled?"
+  type        = bool
+  default     = false
```

Comment:
Should it not be enabled by default? 🤔 Any particular reason why it's not?

Reply:
Agreed, the reason is the same as the ones above, I'll set it to `true` 😁